### DISCUSSION TITLE
Loop start point during playing

### DISCUSSION THREAD

**Discussion by bukru** [bangcorrupt/hacktribe#302] [Ideas]
For now if you made a samples with loop start point defined before loop stop point you have a loopsample that work like this when holding trigger :

Start at begining OR after depending on "EDIT OSC" knob position.
Play to the end of sample even if changing "gate lenght" knob (left to édit FX knob when "amp EG" is off)
Loop back to sample loop start
Play to the end of sample ....
Etc...

The Idea is that if "EDIT OSC" knob is AFTER loopstart point, it become the loop start point.
AND that "GATE LEGHT" knob keep control of the end of samples even if the sample is a looped one.

This allow some really usefull effect for transition in some kind of music such as trance !

**Comment by enorrmann** [comment]
Big if possible, this would allow wavetable osc

**Comment by bangcorrupt** [comment]
This is similar to what's discussed in #58.  I haven't looked into it at all yet.

**Reply by bukru** [reply]
I read this thread, it's similar for sure but i think it's simplier too.

Thanks for reply
